### ISSUE TITLE
Showkeybinding error

### ISSUE THREAD

**Issue by vr8hub** [PowerShell/PSReadLine#588]
Environment data
----------------
PS version: 5.1.16299.19
PSReadline version: 1.2
os: 10.0.16299.15 (WinBuild.160101.0800)
PS file version: 10.0.16299.15 (WinBuild.160101.0800)

Steps to Reproduce
--------------------
Upon initial start of PS, the key for ShowKeyBindings (Ctrl-Alt-?) works fine, i.e. lists all the key bindings as it should.

If I then define this function and key handler, Ctrl-Alt-? starts throwing an error. It doesn't throw the error until the key handler is defined; the function alone doesn't do it.

```
# function to insert and execute a cmdlet string; lets PSR key handler call cmdlet
function PSRInsCmdlet {
    param([string]$cmdlet)

    [Microsoft.PowerShell.PSConsoleReadLine]::RevertLine()
    [Microsoft.PowerShell.PSConsoleReadLine]::Insert($cmdlet)
    [Microsoft.PowerShell.PSConsoleReadLine]::AcceptLine()
}

Set-PSReadLineKeyHandler -Key Alt+E -ScriptBlock {PSRInsCmdlet('exit')}
```

The Alt-E key works, i.e. the PS session closes as expected. It's just ShowKeyBindings that throws the error.

Here is the error from a brand-new session. I started PS (no profile), ran the above commands to define the function and define the key handler, and then immediately hit Ctrl-Alt-?.
```
Oops, something went wrong.  Please report this bug with the details below.
Report on GitHub: https://github.com/lzybkr/PSReadLine/issues/new
-----------------------------------------------------------------------
Last 3 Keys:
 Ctrl+v Enter
 Ctrl+Alt+?

Exception:
System.NullReferenceException: Object reference not set to an instance of an object.
   at Microsoft.PowerShell.PSConsoleReadLine.ShowKeyBindings(Nullable`1 key, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)
-----------------------------------------------------------------------
```

Do I have something wrong in the definition of either the function or the key handler, or is this exposing a problem in ShowKeyBindings?

**Comment by lzybkr** [comment]
Thanks for the report - this is a duplicate of #505.

**Comment by vr8hub** [comment]
Ah, OK. I searched through the open tickets but didn’t think to look through all of the closed ones. My bad.

And I’m sure this is a stupid question, but I’m a bit of a GitHub novice — is the 2.0 pre-release packaged somewhere, or do we have to build it from source to test it?

**Comment by lzybkr** [comment]
Appveyor will always have the most recent build, the link says "build passing" on the main page of the repo - or you can build it yourself.

And no worries about reporting a duplicate - it's always nice if you find it yourself, but sometimes they are hard to find.

An official pre-release will be coming very soon, maybe this weekend.

**Comment by vr8hub** [comment]
Well, I know even less about Appveyor than GitHub. As in nothing. :) When I logged in with my GitHub account, it wanted to set up my projects, but of course this isn’t my project… but never mind, this isn’t Appveyor support. :)

I googled around, found the PSReadline build page in Appveyor, found the artifacts page that had the build zip file on it, and installed it.
All of that is to say, I can confirm that 2.0 fixes the problem — and I like the groupings, especially showing the user-defined functions all together (if 1.2 did that I don’t remember it).

Thanks for your help!
